Mage/src/main/java/mage/abilities/effects/ContinuousEffectsList.java

```diff
@@ -156,6 +156,7 @@ private boolean isInactive(T effect, Game game) {
                     case UntilEndOfYourNextTurn:
                     case UntilEndCombatOfYourNextTurn:
                     case UntilYourNextEndStep:
+                    case UntilTheNextEndStep:
```

Comment:
Same:

![IMG_0868](https://github.com/user-attachments/assets/17187aa2-60d5-4f73-a73c-437230752e1f)

Reply:
Same

Reply:
This section already contains not-'your' durations (custom, for example) so I'm leaving it.